editFunc.js

```diff
@@ -0,0 +1,25 @@
+/* eslint-disable import/prefer-default-export */
+/* eslint-disable eqeqeq */
```

Comment:
[Optional] Kindly remove all disabled Linters, as it helps in writing more professional code

Please check for other occurrences 👍